examples/tutorials/Introduction_To_Material_Science.ipynb

```diff
@@ -1,427 +1,727 @@
 {
```

Comment:
<p>Let's add a loss curve and a comparison to the original CGCNN paper here for reference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3832/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>